README.md

```diff
@@ -1,8 +1,8 @@
 <p align="center">
-    <a href="https://appwrite.io" target="_blank"><img width="260" height="39" src="https://appwrite.io/images/github-logo.png" alt="Appwrite Logo"></a>
+    <a href="https://appwrite.io" target="_blank"><img width="260px" height="39px" src="https://appwrite.io/images/github-logo.png" alt="Appwrite Logo"></a>
```

Comment:
No need for px when used as an HTML attribute, only when used in CSS

Reply:
> No need for px when used as an HTML attribute, only when used in CSS

Okay sir. Thank you.